academic-services/includes/ma-subscription-receive-data.md

```diff
@@ -0,0 +1,96 @@
+Details step-by-step instructions for receiving Microsoft Academic data using Azure Data Share.
+
+## Prerequisites
+
+Before you can accept Azure Data Share invitation to receive Microsoft Academic data, you must provision a number of Azure resources below. Ensure that all pre-requisites are complete before accepting a data share invitation.
+
+* Set up provisioning of Microsoft Academic Graph to an Azure blob storage account. See [Get Microsoft Academic Graph on Azure storage](get-started-setup-provisioning).
+* A Data Share invitation: An invitation from Microsoft Azure with a subject titled "Azure Data Share invitation from **<yourdataprovider@domain.com>**.
```

Comment:
is 'from' the right wording here?  Why would the user get an invitation from themselves?